client/components/forms/counted-textarea/test/index.jsx

```diff
@@ -63,45 +54,50 @@ describe( 'index', function() {
 	} );
 
 	it( 'should not use the placeholder as the counted item if value is empty and countPlaceholderLength is not set', function() {
-		var value = '',
-			placeholder = 'placeholder test',
-			result;
+		const value = '';
+		const placeholder = 'placeholder test';
 
 		renderer.render( <CountedTextarea value={ value } className="custom-class" placeholder={ placeholder } /> );
-		result = renderer.getRenderOutput();
+		const result = renderer.getRenderOutput();
 
 		expect( result.props.children[ 1 ].props.children[ 0 ] ).to.equal( '0 characters' );
 	} );
 
 	it( 'should use the placeholder as the counted item if value is empty and countPlaceholderLength is true', function() {
-		var value = '',
-			placeholder = 'placeholder test',
-			result;
+		const value = '';
+		const placeholder = 'placeholder test';
 
-		renderer.render( <CountedTextarea value={ value } className="custom-class" placeholder={ placeholder } countPlaceholderLength={ true } /> );
-		result = renderer.getRenderOutput();
+		renderer.render(
+			<CountedTextarea value={ value }
+				className="custom-class"
+				placeholder={ placeholder }
+				countPlaceholderLength={ true } />
```

Comment:
> Also, personally I prefer closing those ` />` on the next line, and aligning them vertically with the opening `<CountedTextarea` - feels much more readable to me. It's your call really, but if you decide to go that way, you should probably update the rest of the PR (I've seen several more of these).

Mhm, so far, I've kept the closing `/>` on the same line as the last prop, but you're making a good point about legibility. Anyway, I'll opt to stick as-is due to PR size 😊